benchmarks/kubectl-mtb/hack/ci-test.sh

```diff
@@ -17,5 +17,12 @@ export PATH=$(go env GOPATH)/bin:$PATH
 mkdir -p $(go env GOPATH)/bin
 hack_dir=$(dirname ${BASH_SOURCE})
 
-echo "Running 'make test'"
-make test -C "$hack_dir/.."
\ No newline at end of file
+# install and setup kind according to https://kind.sigs.k8s.io/docs/user/quick-start/
+echo "Setting up Kind"
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.8.1/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /usr/local/bin/kind
+kind create cluster --name kubectl-mtb-suite
```

Comment:
I'd suggest adding a `kind delete cluster --name kubectl-mtb-suite || true` first to ensure you are always creating a new cluster.